server/config.go

```diff
@@ -121,3 +181,57 @@ func (s *Service) ReplaceConfigSection(w http.ResponseWriter, r *http.Request) {
 
 	encodeJSON(w, http.StatusOK, res, s.Logger)
 }
+
+func validLogViewerUIConfig(cfg chronograf.LogViewerUIConfig) error {
+	if len(cfg.Columns) == 0 {
+		return fmt.Errorf("Invalid log viewer UI config: must have at least 1 column")
```

Comment:
Why?

Reply:
Columns can't actually be deleted, only show/hidden, so there's always exactly 9

Reply:
Hmm, this makes me think it might be worthwhile to have a slightly different data structure for columns. Thoughts?

Reply:
such as not using arrays, in order to make sure that there's only 1 of something?, etc?